README.md

```diff
@@ -24,14 +16,44 @@ Its meetings and membership are largely focused on the direction, design, and wo
 
 The following subprojects are owned by the Working Group:
 
-{{
-
-* template-project
-  * Description: Brief description of project. Remove this item and add new projects using this format.
+* Organization Infrastructure
+  * Description: Repositories to manage the working group itself - these are not products for users
+  * Repositories:
+    * https://github.com/RobotWebTools/community
+    * https://github.com/RobotWebTools/robotwebtools.github.io
+    * https://github.com/RobotWebTools/starter-template
+    * https://github.com/RobotWebTools/rwt_dockers
+    * https://github.com/RobotWebTools/robot_web_tools
+* `roslibjs`
+  * Description: ROS 1 Javascript language client - for use in the browser
+  * Repositories:
+    * https://github.com/RobotWebTools/roslibjs
+    * https://github.com/RobotWebTools/roslibjs_integration_tests
+* `rclnodejs`
+  * Description: ROS 2 native NodeJS client
   * Repositories
-    * link-to-repository
-
-}}
+    * https://github.com/RobotWebTools/rclnodejs
+    * https://github.com/RobotWebTools/rclnodejs-cli
+* Rosbridge protocol
+  * Description: A protocol and tools for passing ROS messages to and from web applications
+  * Repositories:
+    * https://github.com/RobotWebTools/ros2-web-bridge
+    * https://github.com/RobotWebTools/rosbridge_suite
+* Visualization components
+  * Description: Common web components/libraries for building ROS visualizers
+  * Repositories:
+    * https://github.com/RobotWebTools/ros3djs
+    * https://github.com/RobotWebTools/ros2djs
+    * https://github.com/RobotWebTools/interactive_marker_proxy
+* Specific Data Streaming
+  * Description: Libraries for streaming ROS data effectively
+  * Repositories:
+    * https://github.com/RobotWebTools/web_video_server
+    * https://github.com/RobotWebTools/tf2_web_republisher
+    * https://github.com/RobotWebTools/webrtc_ros
+    * https://github.com/RobotWebTools/ros_web_video
+    * https://github.com/RobotWebTools/mjpeg_server
```

Comment:
This is deprecated: perhaps we should remove it or at least archive the repository

Reply:
webrtc_ros and tf2_webrepublisher should be under control. ros_web_video and mjpeg_server can be archived. web_video_server is the successor.